pkg/blackduck/ctl_blackduck.go

```diff
@@ -230,6 +233,7 @@ func (ctl *HelmValuesFromCobraFlags) AddCobraFlagsToCommand(cmd *cobra.Command,
 	// Extra Config Settings
 	cmd.Flags().StringVar(&ctl.flagTree.NodeAffinityFilePath, "node-affinity-file-path", defaults.NodeAffinityFilePath, "Absolute path to a file containing a list of node affinities")
 	cmd.Flags().StringVar(&ctl.flagTree.SecurityContextFilePath, "security-context-file-path", defaults.SecurityContextFilePath, "Absolute path to a file containing a map of pod names to security contexts runAsUser, fsGroup, and runAsGroup")
+	cmd.Flags().BoolVar(&ctl.flagTree.IsAzure, "is-azure", defaults.IsAzure, "Whether to deploy in Azure")
```

Comment:
Small comment :) I'm not sure what isAzure does exactly but I'm guessing it configures the deployment for Azure. 

Suggestion: Rephrase the description to say something like "If true, deployment will be configured for Azure"

Similar to [here](https://github.com/blackducksoftware/synopsysctl/pull/128/files#diff-6370acd5ceec69b41b62f02086b17e7399ba63e437cf4201701d50a1896081ecR231)

Reply:
fixed, thanks